src/server/keepalive.js

```diff
@@ -1,5 +1,6 @@
 module.exports = function (client, server, {
   kickTimeout = 30 * 1000,
+  timeoutReason = 'Timed out',
```

Comment:
This should default to keep alive timeout

Reply:
no

Reply:
> **It may be unclear for the user (who sees this text) what this means.**

Reply:
+ it is the vanilla mc message for kick of client that doesnt repond to keepalive packets